src/less/documentation-swedbankpay.less

```diff
@@ -682,9 +686,43 @@
     }
 
     .caption-text {
+        margin-top: 1rem;
         font-style: italic;
         font-size: 0.875rem;
     }
+
+    .color-group {
+        display: flex;
+        gap: 1rem;
```

Comment:
Seems like `gap` is not supported by many browsers (https://caniuse.com/?search=gap). Should be replaced by a more widely supported solution